benchmarks/README.md

```diff
@@ -1,115 +1,136 @@
 # WABuR Benchmarks
 
-**TBD This needs updating to describe the latest approach to creating a project.**
+![](../pages/setup.svg)
 
+![](../pages/throughput.svg)
 
-Using the OpO Runner, a Ruby HTTP client is not able to generate requests and
-process responses quick enough to reach the limits of the Runner. Instead a C
-based HTTP benchmarking tool is used. It is in the OpO download and is called
-**`hose`**.
+![](../pages/latency.svg)
 
-To run the benchmarks start **`opod`** with the `opo/bench.conf` file. This
-turns off the verbosity on `opod` and on the sample Ruby app.
+| Runner        | Read Throughput | Read Latency | Create Throughput | Create Latency | Setup time |
+| ------------- | --------------- | ------------ | ----------------- | -------------- | ---------- |
+| WAB Pure Ruby | 2.8K Reads/sec  | 1.4 msecs    | 2.2K Creates/sec  | 1.8 msecs      |  1 minute  |
+| OpO-Rub       | 212K Reads/sec  | 0.09 msecs   | 134K Creates/sec  | 0.15 msecs     |  1 minute  |
+| OpO Direct    | 364K Reads/sec  | 0.05 msecs   | 157K Creates/sec  | 0.13 msecs     |  1 minute  |
+| Ruby on Rails |  123 Reads/sec  | 175 msecs    | ---- Creates/sec  | ---- msecs     | 20 minutes |
+| Sinatra       | 1.5K Reads/sec  |  13 msecs    | ---- Creates/sec  | ---- msecs     | 60 minutes |
```

Comment:
Sinatra took an hour to set up?

Reply:
Sinatra gives you nothing other than routes and ERB. It has no storage. You have to set up your own separate pages for view, create, edit, etc. If you want a simple REST like API Sinatra does not provide any of that.